site/blog/starlingx-meetup-september-2019.md

```diff
@@ -0,0 +1,48 @@
+
+**"Edge platform for smart agriculture":**
+This case study was quite interesting and attracted a big portion of the audience, not only because of its vivid example (of cultivating shrimps), but more about the realistic difficulties of landing the edge platform in fields (around shrimp ponds). The presenter from 99Cloud were explaining how they took advantage of StarlingX capabilities to address these challenges. Several features were highlighted during the talk, like 'All-in-One Simplex' which is a deployment scenario that StarlingX supports which lets you install the platform only on one server. StarlingX can easily scale from that deployment to a multi-server platform depending on the needs of the use case. The Distributed Cloud feature was also highlighted that they used to manage dozens of hosts with great experience.
+
+**"MEC (Multi-access Edge Computing) platform and implementation by China Unicom Research Institute":**
+The definition of MEC environments has been driven by standards bodies such as 3GPP and ETSI. Following the guidelines of the ETSI MEC architectural framework, China Unicom proposed their reference design on several layers: OSS/BSS, MEAO(MEC Application Orchestrator)/NFVO, VNFM, and MEPM(Multi-access Edge Platform Manager)/MEP(Multi-access Edge Platform). China Unicom chose StarlingX as infrastructure solution for the virtualization layer for its Edge Business Platform to fulfill the requirements on high bandwidth, low latency and massive scale of connections. The presenter also mentioned the importance of StarlingX being fully aligned with ETSI NFV guidelines besides the characteristics that were designed with full dedication to edge use cases. 
+
+**"Edge computing research to empower financial applications by China UnionPay and Topic":**
+By nature, financial applications (for instance, payment use cases) have special requirements, such as increased security and privacy, dynamic and fragmented workloads, and so forth. During this session, the audience could learn about China UnionPay's official designated vendor to Chinese banks) vision and technical approaches how they would utilize edge computing platform to empower typical financial applications. StarlingX was used in a proof of concept deployment of a contactless payment system for electronic-vehicle charging stations as the secure and scalable edge platform to host video surveillance workloads, charge metering and secure payment modules. In the context of deploying financial applications on the edge, StarlingX is an outstanding solution benefiting from its hardened security design (specifically with the well integrated TPM framework) that is backed by security technologies like Intel SGX on Intel platforms.
```

Comment:
By nature, financial applications (for instance, payment use cases) have special requirements, such as increased security and privacy, dynamic and fragmented workloads, and so forth. During this session, the audience could learn about China UnionPay's official designated vendor to Chinese banks vision and technical approaches how they would utilize edge computing platform to empower typical financial applications. StarlingX was used in a proof of concept deployment of a contactless payment system for electronic-vehicle charging stations as the secure and scalable edge platform to host video surveillance workloads, charge metering and secure payment modules. In the context of deploying financial applications on the edge, StarlingX is an outstanding solution benefiting from its hardened security design (specifically with the well integrated TPM framework) that is backed by security technologies like Intel SGX on Intel platforms.